proof/refine/ARM/Schedule_R.thy

```diff
@@ -8,6 +8,9 @@ theory Schedule_R
 imports SchedContext_R InterruptAcc_R
 begin
 
+crunches scReleased
+  for inv: P
```

Comment:
can this be squashed with other `inv` crunches?

Reply:
Actually, I just saw that there is another `crunches` for `inv` lower in the file. I'll move those to the top of this file